src/qt/peertablemodel.cpp

```diff
@@ -115,7 +115,7 @@ PeerTableModel::PeerTableModel(ClientModel *parent) :
     clientModel(parent),
     timer(0)
 {
-    columns << tr("Address/Hostname") << tr("User Agent") << tr("Ping Time");
+    columns << tr("Node/Service") << tr("User Agent") << tr("Ping Time");
```

Comment:
"Address" can be easily confused with bitcoin addresses, and it never displays a hostname. It does, however, display the service (port).